Makefile

```diff
@@ -29,7 +29,8 @@ $(crd_ref_docs_bin): | $(go_bin)
 
 clone-crds:
 	rm -rf .work/crds
-	git clone --depth 1 https://github.com/vshn/component-appcat/ .work/crds
+	git clone --depth 1 --no-single-branch https://github.com/vshn/component-appcat/ .work/crds
+	cd .work/crds && git checkout feature/psql_resource
```

Comment:
Don't forget to remove this before merging

Reply:
Oh that was commited by accident. Removed it again